core/templates/core/base.html

```diff
@@ -3,53 +3,111 @@
 {% load settings_tags %}
 
 {% block body %}
-<nav class="navbar navbar-expand-sm bg-dark navbar-dark">
-    <a class="navbar-brand" href="{% url 'index' %}">Metropolis</a>
-    <ul class="navbar-nav">
-        {% settings_value "NAVBAR" as navbar %}
-        {% for heading, items in navbar.items %}
-        {% if items|gettype == 'str' %}
-        <li class="nav-item">
-            <a class="nav-link" href="{{ items }}">{{ heading }}</a>
-        </li>
-        {% elif items|gettype == 'dict' %}
-        <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">{{ heading }}</a>
-            <div class="dropdown-menu">
-                {% for item, path in items.items %}
-                <a class="dropdown-item" href="{{ path }}">{{ item }}</a>
-                {% endfor %}
-            </div>
-        </li>
-        {% else %}
-        {{ items|gettype }}
-        {% endif %}
+<ul id="dropdownAcc" class="dropdown-content">
+    {% if request.user.is_authenticated %}
+    <li><a href="{% url 'profile_redirect' %}">Profile</a></li>
+    {% if user.is_staff %}
+    <li><a href="{% url 'admin:index' %}">Admin</a></li>
+    {% endif %}
+    <li><a href="{% url 'account_logout' %}">Logout</a></li>
+    {% else %}
+    <li><a href="{% url 'account_login' %}?next={{ request.get_full_path }}">Login</a></li>
+    <li><a href="{% url 'account_signup' %}?next={{ request.get_full_path }}">Register</a></li>
+    {% endif %}
+    <li><a>hi</a></li>
+</ul>
+<ul class="sidenav" id="mobile-demo">
+    {% settings_value "NAVBAR" as navbar %}
+    {% for heading, items in navbar.items %}
+    {% if items|gettype == 'str' %}
+    <li >
+        <a href="{{ items }}">{{ heading }}</a>
+    </li>
+    {% elif items|gettype == 'dict' %}
+    <li class="divider"></li>
+    <li>
+        {% for item, path in items.items %}
+            <li>
+                <a href="#!">{{ item }}</a>
+            </li>
         {% endfor %}
-        <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
+    </li>
+    {% else %}
+    {{ items|gettype }}
+    {% endif %}
+    {% endfor %}
+    <li class="divider"></li>
+    {% if request.user.is_authenticated %}
+    <li><a href="{% url 'profile_redirect' %}">Profile</a></li>
+    {% if user.is_staff %}
+    <li><a href="{% url 'admin:index' %}">Admin</a></li>
+    {% endif %}
+    <li><a href="{% url 'account_logout' %}">Logout</a></li>
+    {% else %}
+    <li><a href="{% url 'account_login' %}?next={{ request.get_full_path }}">Login</a></li>
+    <li><a href="{% url 'account_signup' %}?next={{ request.get_full_path }}">Register</a></li>
+    {% endif %}
+</ul>
+<nav>
+    <div class="nav-wrapper">
+        <a class="brand-logo" href="{% url 'index' %}">Metropolis</a>
+        <a href="#" data-target="mobile-demo" class="sidenav-trigger"><i class="material-icons">menu</i></a>
```

Comment:
```suggestion
        <a href="#" data-target="mobile-demo" class="sidenav-trigger" aria-label="Open navigation menu"><i class="material-icons" aria-hidden="true">menu</i></a>
```

You should do similar for other uses of interactive icons. For all icons in general, I like to follow [this guide from FontAwesome on accessibility](https://fontawesome.com/v4.7/accessibility/).

Reply:
ty for the guide - will keep in mind